src/lighteval/tasks/multilingual/tasks.py

```diff
@@ -1112,6 +1198,1270 @@
+    "elementary_mathematics",
+    "ethnology",
+    "food_science",
+    "genetics",
+    "global_facts",
+    "high_school_biology",
+    "high_school_chemistry",
+    "high_school_geography",
+    "high_school_mathematics",
+    "high_school_physics",
+    "high_school_politics",
+    "human_sexuality",
+    "international_law",
+    "journalism",
+    "jurisprudence",
+    "legal_and_moral_basis",
+    "logical",
+    "machine_learning",
+    "management",
+    "marketing",
+    "marxist_theory",
+    "modern_chinese",
+    "nutrition",
+    "philosophy",
+    "professional_accounting",
+    "professional_law",
+    "professional_medicine",
+    "professional_psychology",
+    "public_relations",
+    "security_study",
+    "sociology",
+    "sports_science",
+    "traditional_chinese_medicine",
+    "virology",
+    "world_history",
+    "world_religions",
+]
+
+cmmlu_tasks = [
+    LightevalTaskConfig(
+        name=f"cmmlu_{Language.CHINESE.value}_{formulation.name.lower()}:{subset}",
+        prompt_function=get_mcq_prompt_function(
+            Language.CHINESE,
+            lambda line: {
+                "question": line["Question"],
+                "choices": [line["A"], line["B"], line["C"], line["D"]],
+                "gold_idx": LETTER_INDICES.index(line["Answer"]),
+            },
+        ),
+        suite=("lighteval",),
+        hf_repo="haonan-li/cmmlu",
+        hf_subset=subset,
+        evaluation_splits=("test",),
+        few_shots_split="dev",
+        metric=(loglikelihood_acc_metric(normalization=LogProbTokenNorm()),),
+    )
+    for subset in CMMLU_SUBSETS
+    for formulation in [
+        MCFFormulation(),
+        CFFormulation(),
+        HybridFormulation(),
+    ]
+]
+
+# Arabic MMLU: Arabic version of MMLU
+# Native translation with some new categories
+# Paper: https://arxiv.org/html/2402.12840v1
+ARABIC_MMLU_SUBSETS = [
+    "Driving Test",
+    "High Geography",
+    "High History",
+    "Islamic Studies",
+    "Univ Accounting",
+    "Primary General Knowledge",
+    "Univ Political Science",
+    "Primary Math",
+    "Middle General Knowledge",
+    "High Biology",
+    "Primary Natural Science",
+    "High Economics",
+    "Middle Natural Science",
+    "Middle Geography",
+    "Primary Social Science",
+    "Middle Computer Science",
+    "Middle Islamic Studies",
+    "Primary Computer Science",
+    "High Physics",
+    "Middle Social Science",
+    "Middle Civics",
+    "High Computer Science",
+    "General Knowledge",
+    "High Civics",
+    "Prof Law",
+    "High Islamic Studies",
+    "Primary Arabic Language",
+    "High Arabic Language",
+    "Arabic Language (Grammar)",
+    "Primary History",
+    "Middle History",
+    "Univ Economics",
+    "Arabic Language (General)",
+    "Univ Computer Science",
+    "Primary Islamic Studies",
+    "Primary Geography",
+    "High Philosophy",
+    "Middle Arabic Language",
+    "Middle Economics",
+    "Univ Management",
+]
+
+arabic_mmlu_tasks = [
```

Comment:
I think we already have it in a community task

Reply:
There are some redefinitions from alghafa that's true. That's because the community tasks don't use templates and don't allow to easily switch between mcf/cf.

Altought this exact tasks is not from alghafa. It's native mmlu (unlike the one from alghafa).

Reply:
ok sounds good !